src/TicketingSystem.Api/Filters/BusinessLogicExceptionFilter.cs

```diff
@@ -9,16 +9,26 @@ public class BusinessLogicExceptionFilterAttribute : ExceptionFilterAttribute
     {
         public override void OnException(ExceptionContext context)
         {
-            if(context.Exception is BusinessLogicException bex && bex.Code == Common.Enums.ErrorCode.NotFound)
+            if(context.Exception is BusinessLogicException bex)
             {
+                int statusCode;
+                if(bex.Code == Common.Enums.ErrorCode.NotFound)
+                {
+                    statusCode = (int)HttpStatusCode.NotFound;
+                }
+                else
+                {
+                    statusCode = (int)HttpStatusCode.BadRequest;
+                }
```

Comment:
Can be simplified:
```
int statusCode = bex.Code == Common.Enums.ErrorCode.NotFound
? (int)HttpStatusCode.NotFound
: (int)HttpStatusCode.BadRequest;
```
But this is probably a matter of choice

Reply:
That was for the cases of other options but NotFound & BadRequest, but it's true that for now it can be simplified. Updated